adapters/marsmedia/usersync.go

```diff
@@ -0,0 +1,12 @@
+package marsmedia
+
+import (
+	"text/template"
+
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/usersync"
+)
+
+func NewMarsmediaSyncer(temp *template.Template) usersync.Usersyncer {
+	return adapters.NewSyncer("marsmedia", 0, temp, adapters.SyncTypeRedirect)
```

Comment:
The `0` argument is for the GDPR vendorID. Does MarsMedia have a GDPR vendor id?

Reply:
No.